blueocean-dashboard/src/main/resources/jenkins/plugins/blueocean/dashboard/Messages_ru.properties

```diff
@@ -0,0 +1,169 @@
+#X-Generator: crowdin.com
+# common
+common.date.duration.format=d[d] h[h] m[m] s[s]
+common.date.duration.hint.format=M [month], d [days], h[ hours], m[ minutes], s[ seconds]
+common.date.duration.display.format=d[d] h[h] m[m] s[s]
+common.date.readable.long=MMM DD YYYY h\:mma Z
+common.date.readable.short=MMM DD h\:mma Z
+common.pager.loading=\u0417\u0430\u0433\u0440\u0443\u0437\u043a\u0430...
+common.pager.more=\u041f\u043e\u043a\u0430\u0437\u0430\u0442\u044c \u0431\u043e\u043b\u044c\u0448\u0435
+# creation
+creation.core.header.close=\u0417\u0430\u043a\u0440\u044b\u0442\u044c
+creation.core.header.title=\u0421\u043e\u0437\u0434\u0430\u0442\u044c \u041a\u043e\u043d\u0432\u0435\u0439\u0435\u0440
+creation.core.header.classic=\u041f\u0435\u0440\u0435\u0439\u0442\u0438 \u043a \u043a\u043b\u0430\u0441\u0441\u0438\u0447\u0435\u0441\u043a\u043e\u043c\u0443 \u0441\u043e\u0437\u0434\u0430\u043d\u0438\u044e \u044d\u043b\u0435\u043c\u0435\u043d\u0442\u043e\u0432
+creation.core.intro.invalid_permission_title=\u0423 \u0432\u0430\u0441 \u043d\u0435\u0442 \u0440\u0430\u0437\u0440\u0435\u0448\u0435\u043d\u0438\u0439 \u043d\u0430 \u0441\u043e\u0437\u0434\u0430\u043d\u0438\u0435 \u043a\u043e\u043d\u0432\u0435\u0439\u0435\u0440\u043e\u0432
+creation.core.intro.invalid_permission_button=\u0412\u0435\u0440\u043d\u0443\u0442\u044c\u0441\u044f \u043a \u041a\u043e\u043d\u0432\u0435\u0439\u0435\u0440\u0430\u043c
+creation.core.intro.invalid_security_linkhref=https\://wiki.jenkins-ci.org/display/JENKINS/Securing+Jenkins
```

Comment:
Remove - no difference from the Messages.properties. @i386  the recommended way is to create redirects from jenkins.io. CC @daniel-beck